mne/annotations.py

```diff
@@ -274,10 +323,12 @@ class Annotations:
     :meth:`Raw.save() <mne.io.Raw.save>` notes for details.
     """  # noqa: E501
 
-    def __init__(self, onset, duration, description, orig_time=None, ch_names=None):
+    def __init__(
+        self, onset, duration, description, orig_time=None, ch_names=None, details=None
```

Comment:
```suggestion
        self, onset, duration, description, orig_time=None, ch_names=None, extras=None
```
thoughts my native english speaker friends?